ci/ci.yml

```diff
@@ -214,7 +214,7 @@ Resources:
             Value:
               Fn::ImportValue:
                 !Sub ${InfrastructureStorageStackName}-InfrastructureApplicationCodeBucket
-        Image: aws/codebuild/amazonlinux2-x86_64-standard:1.0
+        Image: aws/codebuild/amazonlinux2-x86_64-standard:3.0
```

Comment:
But most of our repos run docker inside this codebuild container (docker in docker)... so this shouldn't really matter, right?  I can't think of any that run the tests directly in the parent container.

Reply:
Probably just my stuff, like Infrastructure and Porter